djasana/management/commands/sync_from_asana.py

```diff
@@ -271,11 +271,14 @@ def _sync_project_id(self, project_id, workspace, models):
         if Task in models and not project_dict['archived'] or self.process_archived:
             remote_task_ids = []
             for task in self.client.tasks.find_all({'project': project_id}):
-                remote_task_ids.append(task['id'])
-                self._sync_task(task, project, models)
+                ids = self._sync_task(task, project, models)
+                remote_task_ids.extend(ids)
             # Delete local tasks for this project that are no longer in Asana.
-            Task.objects.filter(projects=project).exclude(
-                remote_id__in=remote_task_ids).exclude(remote_id__isnull=True).delete()
+            tasks_to_delete = Task.objects.filter(projects=project).exclude(
+                remote_id__in=remote_task_ids).exclude(remote_id__isnull=True)
+            id_list = list(tasks_to_delete.values_list('remote_id', flat=True))
+            logger.debug("Deleting Tasks No Longer Present " + str(id_list))
```

Comment:
The rest of the module logs to logger and writes to stdout for console feedback; please do that here. 
message = 'Deleting tasks no longer present: {}.'.format(id_list)

Reply:
sounds good, did this update